projects/habitat_ovmm/evaluator.py

```diff
@@ -150,7 +170,8 @@ def stop():
                         **metrics_at_skill_end,
                         **episode_metrics[episode_key],
                     }
-                    episode_metrics[episode_key]["goal_name"] = info["goal_name"]
+                    if "goal_name" in episode_metrics[episode_key]:
```

Comment:
The same question here: 
Why have you added `if "goal_name" in episode_metrics[episode_key]`? There was no such check before (in older evaluator code).

Reply:
This was being set inside ObjectNavAgent and I am not setting this in RandomAgent